### PR TITLE
Initial conversion styling

### PR DESCRIPTION
**Pull request by RakshaPawar108**
- Converted all screens into React
- Gave initial styling to all the pages
- Broke down few pages into smaller components

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]
1.you have defined the folders and file in well structured manner
2.liked the way you split components into smaller chunks
3.good work

File: src/components/Filter/Filter.jsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          <label for="cost-slider">Rs.0 - Rs.1000</label>
+          <input type="range" id="cost-slider" min="0" max="1000" value="500" />
+        </div>
+      </div>
+
+      <div className="filter-type">
+        <h3 className="filter-type-heading">Category</h3>
+        <div className="filter-type-container">
+          <input type="checkbox" id="healthy" />
+          <label for="healthy">Healthy</label>
```
Comment (RahullRawat) [inline src/components/Filter/Filter.jsx]: You can use **htmlFor** attribute instead of for

**Review by RahullRawat — APPROVED** [review]
Apart from suggestion code looks good. Good folder structure.

File: src/components/Filter/Filter.jsx (hunk 1 above)
Comment (RakshaPawar108) [reply src/components/Filter/Filter.jsx]: Didn't understand the reason at first but after doing a bit of reading, I understood why `for` attribute needed to be changed to `htmlFor`. Thanks for helping me learn something new 😄

**Pull request merged**